LiveSplit.SuperMetroid.asl

```diff
@@ -320,6 +348,42 @@ startup
 		{ "chargeBeam",		0x10}
 	};
 
+	vars.missileWatchKeys = new Dictionary<string, string> {
+		{ "alphaMissiles", "brinstarItems4" },
```

Comment:
I assume these hashes are ordered by how they're usually picked up in 100%?

I think having them grouped by area, then sorted by bitflag order would be neater.